src/serial/serialcommands.cpp

```diff
@@ -203,9 +203,10 @@ void printState() {
 		}
 	}
 	logger.info(
-		"Battery voltage: %.3f, level: %.1f%%",
+		"Battery voltage: %.3f, level: %.1f%%, %s",
```

Comment:
this adds an unnecessary leading "," when anything other than the BQ chip is in use

Reply:
Fixed